t/t3433-rebase-options-compatibility.sh

```diff
@@ -84,4 +84,22 @@ test_expect_success '--committer-date-is-author-date works with interactive back
 	test_cmp authortime committertime
 '
 
+# Checking for +0000 in author time is enough since default
+# timezone is UTC, but the timezone used while committing
+# sets to +0530.
+test_expect_success '--ignore-date works with am backend' '
+	git commit --amend --date="$GIT_AUTHOR_DATE" &&
+	git rebase --ignore-date HEAD^ &&
+	git cat-file commit HEAD | sed -e "/^\$/q" >head &&
```

Comment:
How about using `git log --pretty="format:%ai"` instead?

Reply:
Yeah, I think that would be preferrable.